example_test.go

```diff
@@ -311,3 +343,34 @@ func Example_copyArtifactManifestRemoteToLocal() {
 	// Output:
 	// true
 }
+
+func Example_copyArtifactManifestAndReferrersRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/source", remoteHost))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+	ctx := context.Background()
+
+	// tag exampleManifest as latest in the remote repository
+	exampleTag := "latest"
+	err = src.Tag(ctx, exampleManifestDescriptor, exampleTag)
+	if err != nil {
+		panic(err)
+	}
+	// copy this manifest and its referrer(exampleSignatureManifest) using ExtendedCopy
+	copiedDescriptor, err := oras.ExtendedCopy(ctx, src, exampleTag, dst, exampleTag, oras.DefaultExtendedCopyOptions)
+	if err != nil {
+		panic(err)
+	}
+	// verify that exampleManifest and its referrer exist in dst
+	fmt.Println(reflect.DeepEqual(copiedDescriptor, exampleManifestDescriptor))
+	manifestExists, _ := dst.Exists(ctx, exampleManifestDescriptor)
+	fmt.Println(manifestExists)
+	referrerExists, _ := dst.Exists(ctx, exampleSignatureManifestDescriptor)
```

Comment:
We may not need to check this, as it looks like the `exampleSignatureManifestDescriptor` comes from nowhere. @qweeah Any suggestions?

Reply:
Then how can I verify that the referrers are copied? Or I just don't need to check?

Reply:
From our discussion we decided that we don't need to check it.